omnibus/files/private-chef-cookbooks/private-chef/recipes/default.rb

```diff
@@ -50,6 +50,12 @@
 include_recipe 'private-chef::plugin_config_extensions'
 include_recipe 'private-chef::config'
 
+if node['private_chef']['elasticsearch']['first_internal_install']
```

Comment:
So there's no pre-defined default for this, we're relying on nil being 'falsey', right?

Reply:
added to the default attributes, defaults to false.